website/docs/patterns/form-patterns/partials/validation/validation.md

```diff
@@ -1,40 +1,37 @@
-!!! Warning
-
-While we provide the structure and visual consistency for validation, the messaging and functionality need to be implemented by the product teams.
-!!!
-
-Form validation ensures that fields are correctly formatted, required fields are not empty, and the data being submitted matches the requirements for application. A successful validation strategy relies on three key concepts:
+Form validation ensures that fields are correctly formatted, required fields are not empty, and the data being submitted matches the requirements for the application. A successful validation strategy relies on three key concepts:
 
-1. **Uses clear language:** the validation message should be human readable and easy to understand.
-2. **Displayed at the right time:** The validation message should be displayed as soon as possible to when the error occurred.
-3. **Displayed in the right place:** the validation message should be noticeable and located where the error occurred.
+1. **Clear language:** The validation message should be human-readable and easy to understand.
+2. **Placement:** The validation message should be displayed as soon as possible after the error occurred.
```

Comment:
The explanations for these two seem to be switched:

Placement: The validation message should be displayed as soon as possible after the error occurred.
Timing: The validation message should be noticeable and located where the error occurred.